pkg/env/prod.go

```diff
@@ -46,8 +46,10 @@ type prod struct {
 
 	clusterGenevaLoggingCertificate   *x509.Certificate
 	clusterGenevaLoggingPrivateKey    *rsa.PrivateKey
+	clusterGenevaLoggingAccount       string
 	clusterGenevaLoggingConfigVersion string
-	clusterGenevaLoggingEnvironment   string
+	clusterGenevaLoggingNamespace     string
+	genevaLoggingEnvironment          string
```

Comment:
Same comment as below.  Naming should be consistent _(it references the same env variable)_